dags/probe_scraper.py

```diff
@@ -229,22 +231,15 @@
         **airflow_gke_prod_kwargs,
     )
 
-    probe_scraper_checks = []
-    for check_name, week_day in [("check-expiry", "Monday"), ("check-fog-expiry", "Wednesday")]:
-        day_check = DayOfWeekSensor(
-            task_id=f"week_day_check_{week_day.lower()}",
-            week_day=week_day,
-            use_task_execution_day=True,
-            dag=dag,
-        )
-        check_task = GKEPodOperator(
+    probe_scraper_checks = [
+        GKEPodOperator(
             task_id=f"probe_scraper_{check_name.replace('-', '_')}",
             name=f"probe-scraper-{check_name}",
             image=probe_scraper_image,
             arguments=(
                 probe_scraper_base_arguments
                 + [
-                    "--{check_name}",
```

Comment:
Was this not working before?

Reply:
it was not (was supposed to be an f-string)